daffodil-core/src/main/scala/org/apache/daffodil/core/grammar/ElementBaseGrammarMixin.scala

```diff
@@ -194,8 +194,9 @@ trait ElementBaseGrammarMixin
       )
       schemaDefinitionWhen(
         detachedElementDecl.statements.nonEmpty,
-        "%s is specified as a dfdl:prefixLengthType, but specifies one or more statement annotations",
-        prefixLengthType
+        "%s is specified as a dfdl:prefixLengthType, but specifies one or more statement annotations (%s)",
+        prefixLengthType,
+        detachedElementDecl.statementsAsStrings.mkString(", ")
```

Comment:
I believe `toString` on any DFDLAnnotation already does this conversion to a pretty string, so I don't think we need `statementsAsString`. I *think* you can just do 
```scala
detachedElementDecl.statemtents.mkString(", ")
```
And it'll do the right thing.

Reply:
It does indeed!